package.json

```
@@ -70,15 +74,15 @@
     "react-native-barcode-svg": "^0.0.15",
     "react-native-draggable-flatlist": "^4.0.1",
     "react-native-draglist": "^3.6.1",
-    "react-native-gesture-handler": "^2.17.1",
+    "react-native-gesture-handler": "^2.16.1",
     "react-native-infinite-pager": "^0.3.16",
```

Met à jour vers la dernière version pour résoudre un des bugs ts
```suggestion
    "react-native-infinite-pager": "^0.3.18",
```

---

ça a rien changé mais why not !

---

Ah c'est bizarre, alors ça marchait sur une version < 0.3.16